js/admin.js

```diff
@@ -110,6 +110,7 @@ jQuery( document ).ready(function($) {
 							}
 						}
 					};
+					$('.progress-bar').addClass('install-error');
```

Comment:
I agree that the message should be something along the: 'Check failed due to lack of memory', or just 'Check failed' because we already show the error message on the failed file.